diff -bur XML_Feed_Parser-0.3.0.orig/usr/share/pear/XML/Feed/Parser.php XML_Feed_Parser-0.3.0/usr/share/pear/XML/Feed/Parser.php
--- XML_Feed_Parser-0.3.0.orig/usr/share/pear/XML/Feed/Parser.php	2006-01-12 11:47:22.988618000 +0100
+++ XML_Feed_Parser-0.3.0/usr/share/pear/XML/Feed/Parser.php	2006-01-12 11:48:37.467295520 +0100
@@ -26,12 +26,12 @@
  * feed types. It makes sense to load it here to keep the other files
  * clean.
  */
-require_once 'Parser/Type.php';
+require_once 'XML/Feed/Parser/Type.php';
 
 /**
  * We will throw exceptions when errors occur.
  */
-require_once 'Parser/Exception.php';
+require_once 'XML/Feed/Parser/Exception.php';
 
 /**
  * This is the core of the XML_Feed_Parser package. It identifies feed types 
@@ -108,13 +108,13 @@
 
         switch (true) {
             case ($doc_element->namespaceURI == 'http://www.w3.org/2005/Atom'):
-                require_once 'Parser/Atom.php';
-                require_once 'Parser/AtomElement.php';
+                require_once 'XML/Feed/Parser/Atom.php';
+                require_once 'XML/Feed/Parser/AtomElement.php';
                 $class = 'XML_Feed_Parser_Atom';
                 break;
             case ($doc_element->namespaceURI == 'http://purl.org/atom/ns#'):
-                require_once 'Parser/Atom.php';
-                require_once 'Parser/AtomElement.php';
+                require_once 'XML/Feed/Parser/Atom.php';
+                require_once 'XML/Feed/Parser/AtomElement.php';
                 $class = 'XML_Feed_Parser_Atom';
                 $error = 'Atom 0.3 deprecated, using 1.0 parser which won\'t provide ' .
                     'all options';
@@ -123,40 +123,40 @@
                 ($doc_element->hasChildNodes() and $doc_element->childNodes->length > 1 
                 and $doc_element->childNodes->item(1)->namespaceURI == 
                 'http://purl.org/rss/1.0/')):
-                require_once 'Parser/RSS1.php';
-                require_once 'Parser/RSS1Element.php';
+                require_once 'XML/Feed/Parser/RSS1.php';
+                require_once 'XML/Feed/Parser/RSS1Element.php';
                 $class = 'XML_Feed_Parser_RSS1';
                 break;
             case ($doc_element->namespaceURI == 'http://purl.org/rss/1.1/' or 
                 ($doc_element->hasChildNodes() and $doc_element->childNodes->length > 1 
                 and $doc_element->childNodes->item(1)->namespaceURI == 
                 'http://purl.org/rss/1.1/')):
-                require_once 'Parser/RSS11.php';
-                require_once 'Parser/RSS11Element.php';
+                require_once 'XML/Feed/Parser/RSS11.php';
+                require_once 'XML/Feed/Parser/RSS11Element.php';
                 $class = 'XML_Feed_Parser_RSS11';
                 break;
             case (($doc_element->hasChildNodes() and $doc_element->childNodes->length > 1
                 and $doc_element->childNodes->item(1)->namespaceURI == 
                 'http://my.netscape.com/rdf/simple/0.9/') or 
                 $doc_element->namespaceURI == 'http://my.netscape.com/rdf/simple/0.9/'):
-                require_once 'Parser/RSS09.php';
-                require_once 'Parser/RSS09Element.php';
+                require_once 'XML/Feed/Parser/RSS09.php';
+                require_once 'XML/Feed/Parser/RSS09Element.php';
                 $class = 'XML_Feed_Parser_RSS09';
                 break;
             case ($doc_element->tagName == 'rss' and
                 $doc_element->hasAttribute('version') and 
                 $doc_element->getAttribute('version') == 0.91):
                 $error = 'RSS 0.91 has been superceded by RSS2.0. Using RSS2.0 parser.';
-                require_once 'Parser/RSS2.php';
-                require_once 'Parser/RSS2Element.php';
+                require_once 'XML/Feed/Parser/RSS2.php';
+                require_once 'XML/Feed/Parser/RSS2Element.php';
                 $class = 'XML_Feed_Parser_RSS2';
                 break;
             case ($doc_element->tagName == 'rss' and
                 $doc_element->hasAttribute('version') and 
                 $doc_element->getAttribute('version') == 0.92):
                 $error = 'RSS 0.92 has been superceded by RSS2.0. Using RSS2.0 parser.';
-                require_once 'Parser/RSS2.php';
-                require_once 'Parser/RSS2Element.php';
+                require_once 'XML/Feed/Parser/RSS2.php';
+                require_once 'XML/Feed/Parser/RSS2Element.php';
                 $class = 'XML_Feed_Parser_RSS2';
                 break;
             case (in_array($doc_element->namespaceURI, $this->feedNamespaces['rss2'])
@@ -165,8 +165,8 @@
                     $doc_element->getAttribute('version') != 2) {
                     $error = 'RSS version not specified. Parsing as RSS2.0';
                 }
-                require_once 'Parser/RSS2.php';
-                require_once 'Parser/RSS2Element.php';
+                require_once 'XML/Feed/Parser/RSS2.php';
+                require_once 'XML/Feed/Parser/RSS2Element.php';
                 $class = 'XML_Feed_Parser_RSS2';
                 break;
             default:
Only in XML_Feed_Parser-0.3.0/usr/share/pear/XML/Feed: Parser.php~
